Automattic/wp-calypso
# Jetpack Plans: Add Easy Site Migration to Jetpack Personal Plan feature listing.

#### Chages introduced by this PR

Adds the **Easy Site Migration** item to the list of Jetpack Personal Plan features. 

#### Testing instructions

1. Visit `http://calypso.localhost:3000/plans/{site-slug}` where site is a Jetpack site's slug.
1. Confirm you see `Easy Site Migration` listed among the Jetpack Personal Plan features (Both monthly and yearly).
